scss/literallycanvas.scss

```diff
@@ -82,13 +87,21 @@ $selectedColor: lighten(#3cb0fd, 20%);
     position: absolute;
   }
 
+  // @ifdef INCLUDE_GUI
   .lc-picker {
     top: 0; left: 0; bottom: 0; width: $pickerWidth;
     background-color: $toolbarBackground;
   }
+  // @endif
 
   .lc-drawing {
+    // @ifdef INCLUDE_GUI
     right: 0; left: $pickerWidth;
+    // @endif
+    // @ifndef INCLUDE_GUI
+    right: 0; left: 0; top: 0; bottom: 0;
+    // @endif
```

Comment:
One last thing: instead of needing different CSS files, could you just tweak the CSS so it doesn't behave badly without the GUI? I think all you should need to do is change this block to:

```scss
.lc-drawing.gui {
    /* original contents */
}
```

and change `index.coffee` in a way I will describe below...